### PR TITLE
Feat(Account Settings): Layout and Notifications Panel

### PR DESCRIPTION
**Pull request by Chrissy**
This pull sets up a basic layout for personal settings and introduces a radio button component that matches out boolean component. 

defaults-5
radios

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx
Comment (ckj) [inline src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx]: 😍

**Review by ckj — COMMENTED** [review]
This looks great.  There's way more JS than CSS though, so I'll defer to @billyvg for the final 👍 .

**Comment by Chrissy** [comment]
@ckj thanks!

File: .storybook/webpack.config.js
Comment (billyvg) [inline .storybook/webpack.config.js]: I would additionally create an alias to `path.resolve(staticPath, 'app')` so that we don't have to keep creating aliases and just make everything relative to `app`.

File: src/sentry/static/sentry/app/routes.jsx
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: When this rolls out to GA, are we going to switch to whatever url we're currently using (I think it's `account`?)

File: src/sentry/static/sentry/app/views/settings/components/forms/formField.jsx
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/components/forms/formField.jsx]: 😍

File: tests/js/spec/components/forms/radioField.spec.jsx
Comment (billyvg) [inline tests/js/spec/components/forms/radioField.spec.jsx]: I think `choices` should just be an array, the function doesn't add much here

File: src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx]: Maybe turn this into a `<RadioGroup>` component that takes props. Then I think the tests you were trying to write will become easier since you can test `<RadioGroup>` vs the field that's wrapped with `FormField`

**Review by billyvg — CHANGES_REQUESTED** [review]
Left some suggestions, let me know if you have questions.

I also think we should enforce the feature tag on the routes just because the views are unfinished and it would be a weird experience if they somehow do manage to stumble upon them.

File: src/sentry/static/sentry/app/routes.jsx
Comment (Chrissy) [reply src/sentry/static/sentry/app/routes.jsx]: This is a good question. Two answers:

1. I switched from _account_ to _personal_ because I thought it would be harder to misunderstand, but it looks like @ckj has been using _account_ and the communication is good, so I'll switch the language and naming conventions back to use _account_. 

image

2. That said, I think `account` should be a sub-route of `/settings` just like everything else, so maybe when everything is rtg we should just redirect `/account` urls to `/settings/account` and change the links?

**Comment by billyvg** [comment]
sounds good

File: src/sentry/static/sentry/app/routes.jsx
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: Not sure if you've rebased w/ master but this shouldn't be here.

**Review by billyvg — APPROVED** [review]
There's 2 minor issues I see, but lgtm once those are fixed

**Comment by Chrissy** [comment]
@billyvg ok Travis is taking flippin' forever but js tests ran fine so it should pass in due time. I fixed the things you pointed out so we should be good! will merge first thing tomorrow.

**Comment by billyvg** [comment]
You'll want to run `yarn test -u` to update jest snapshots after you rebase w/ master. There was an enzyme change that adds `key` to jest snapshots.

**Pull request merged**
